venue.html.j2

```diff
@@ -15,6 +15,24 @@
     </small>
   </figure>
 
+  <h2>Hotels</h2>
+
+PGConf.dev has negotiated discounted rates with several hotels in Montreal. We recommend that you book your accommodations early, whether or not you're staying at one of the hotels below, as there are many activities around Montreal at this time of the year.
+
+  <h3>Alt Hotel Montreal</h3>
+  <p>
+  <a href="https://www.germainhotels.com/en/le-germain-hotel/montreal">The Alt Hotel</a> is located
```

Comment:
Let's make sure that we add `target="_blank"` to all external links on the site, including this one and the 4 or so following links.

Reply:
- [X ] Added blank target